### PR TITLE
Misc typo fixes in calicoctl dir

### PR DESCRIPTION
**Pull request by Akasurde**
## Description

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: pod2daemon/flexvol/flexvoldriver.go [hunk 1]
```diff
@@ -307,7 +307,7 @@ func unmount(dir string) error {
 	if len(comps) < 6 {
 		sErr := fmt.Sprintf("Failure to notify nodeagent dir %v", dir)
 		logError("unmount", dir, sErr, syslogOnlyFalse)
-		return fmt.Errorf("invalid path to unount")
+		return fmt.Errorf("invalid path to umount")
```
Comment (neiljerram) [inline pod2daemon/flexvol/flexvoldriver.go:310]: Looks like this one should be "unmount" ?

**Review by neiljerram — CHANGES_REQUESTED** [review]
Just one correction, I think.

**Review by neiljerram — APPROVED** [review]

**Comment by Akasurde** [comment]
@neiljerram @caseydavenport Thanks for the review and merge.

**Pull request merged**